smithy-model/src/main/java/software/amazon/smithy/model/SourceLocation.java

```diff
@@ -97,4 +99,23 @@ public int hashCode() {
 
         return h;
     }
+
+    public static class Sorter implements Comparator<FromSourceLocation>, Serializable {
```

Comment:
Why implement Serializable? Why not add this functionality to SourceLocation directly and implement Comparable?

Reply:
And what about moving this helper method to a static method on`FromSourceLocation` called `compare` to match how stuff like `Integer.compare` already works in Java?